docker/Dockerfile

```diff
@@ -1,21 +1,16 @@
 ARG PHP=8.1
 FROM php:${PHP}-cli-alpine
 
-RUN apk update \
-    && apk add zip libzip-dev icu-dev git
+RUN apk update && apk add \
+    zip libzip-dev icu-dev git \
```

Comment:
Won't this cause issues as it's a trailing newline?

Reply:
It does. 

![Screenshot 2023-12-27 at 7 48 08 PM](https://github.com/pestphp/pest/assets/664857/3d6bda4e-5652-4da3-bfc5-dd78cdc519cf)